src/rules/objectLiteralShorthandRule.ts

```diff
@@ -63,9 +93,16 @@ function walk(ctx: Lint.WalkContext<void>) {
     });
 }
 
-function handleLonghandMethod(name: ts.PropertyName, initializer: ts.FunctionExpression, sourceFile: ts.SourceFile): [string, Lint.Fix] {
+function handleLonghandMethod(
+    name: ts.PropertyName,
+    initializer: ts.FunctionExpression,
+    sourceFile: ts.SourceFile,
+): [string, Lint.Fix] {
```

Comment:
Please avoid all these purely stylistic changes. They don't add any value to the PR and make the diff harder to reason about.

Reply:
I think I have a TS formatter formatting on save. I'll take care of it.